CODEOWNERS

```diff
@@ -2,5 +2,5 @@
 /Makefile*       @istio/wg-test-and-release-maintainers
 /*.md            @istio/wg-test-and-release-maintainers
 /common/         @istio/wg-test-and-release-maintainers
-/common-protos/  @istio/wg-test-and-release-maintainers
+/proto/          @istio/wg-test-and-release-maintainers
```

Comment:
Just something I noticed ... I think this is correct?

Reply:
No, common-protos is just not used here, should be removed if anything. protos/ should belong to ztunnel maintainers

Reply:
done.